dom/src/DOMSourceable.ts

```diff
@@ -0,0 +1,7 @@
+import {Sourceable} from './Sourceable';
+export {GenericStream} from './GenericStream';
+export {EventsFnOptions} from './EventsFnOptions';
+
+export interface DOMSourceable extends Sourceable {
```

Comment:
Short answer: Because `HTMLSource` is also a `Sourceable` with a slightly different signature than `DOMSource`s, the abstract class from which `MainDOMSource`, `DocumentDOMSource`, and `BodyDOMSource` extend from.